layers/+lang/plantuml/README.org

```diff
@@ -0,0 +1,28 @@
+#+TITLE: plantuml layer
+#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../css/readtheorg.css" />
+
+* Table of Contents                                        :TOC_4_org:noexport:
+ - [[Description][Description]]
+ - [[Install][Install]]
+ - [[Key bindings][Key bindings]]
+
+* Description
+This layer enables support for [[%5B%5Bfile:img/logo.png%5D%5D%0Ahttps://github.com/skuro/puml-mode][puml-mode]], which provides
```

Comment:
`%5B` `%5D`?? Is this the github's rendering fault or do you really have this in your file instead of brackets?